### PR TITLE
Smith

### PR DESCRIPTION
**Pull request by SMITH1103**
**Description**:
Submitting previous work to Next

**List of changes**:
-- Work on responsive pages and dark mode
-- Unit Converter
-- DownloadFile.vue in progress

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -39,7 +39,10 @@
 
   <body
     class="
+<<<<<<< HEAD
```
Comment (cacampbell) [inline index.html:42]: Fix

File: locales/en.json [hunk 2]
```diff
@@ -48,6 +49,8 @@
   "InterfaceAsset.note2": "Your asset is frozen.",
   "InterfaceAsset.value": "{value} ea.",
   "InterfaceAssets.header": "Assets",
+  "InterfaceAssets.usd": "USD",
+  "InterfaceDownload.modal.fee": "There may be a fee associated with this transaction.",
```
Comment (cacampbell) [inline locales/en.json:53]: All transactions have fees

File: locales/en.json [hunk 3]
```diff
@@ -106,6 +111,8 @@
   "InterfaceToolTile.associateToken.title": "Associate Token",
   "InterfaceToolTile.associateToken.label.tokenID": "Token ID",
   "InterfaceToolTile.download.title": "Download",
+  "InterfaceToolTile.download.description": "Download a file with a File ID.",
```
Comment (cacampbell) [inline locales/en.json:114]: all files have a file id

File: package.json [hunk 4]
```diff
@@ -40,7 +40,7 @@
     "preferred-locale": "^1.0.10",
     "vue": "^3.0.11",
     "vue-i18n": "^9.1.6",
-    "vue-router": "^4.0.8"
+    "vue-router": "^4.0.10"
```
Comment (cacampbell) [inline package.json:43]: undo this change

File: src/components/base/DropdownSelector.vue [hunk 5]
```diff
@@ -20,7 +20,11 @@
         bg-white
         rounded
         cursor-pointer
+<<<<<<< HEAD
```
Comment (cacampbell) [inline src/components/base/DropdownSelector.vue:23]: Fix

File: src/components/interface/Headline.vue [hunk 6]
```diff
@@ -7,6 +7,10 @@
       font-medium
       dark:text-white
       dark:border-midnight-express
+<<<<<<< HEAD
```
Comment (cacampbell) [inline src/components/interface/Headline.vue:10]: Fix

File: src/components/interface/Modal.vue [hunk 7]
```diff
@@ -32,7 +32,11 @@
           bg-white
           dark:bg-ruined-smores
           text-carbon
+<<<<<<< HEAD
```
Comment (cacampbell) [inline src/components/interface/Modal.vue:35]: Fix

File: src/components/interface/Modal.vue [hunk 8]
```diff
@@ -46,7 +50,11 @@
         ]"
         @click.stop
       >
+<<<<<<< HEAD
```
Comment (cacampbell) [inline src/components/interface/Modal.vue:53]: Fix

File: src/components/interface/Modal.vue [hunk 9]
```diff
@@ -83,6 +91,10 @@ export default defineComponent({
   emits: ["close"],
   setup() {
     return { closeIcon, closeIconDark };
+<<<<<<< HEAD
```
Comment (cacampbell) [inline src/components/interface/Modal.vue:94]: Fix

File: src/components/interface/TransferForm.vue [hunk 10]
```diff
@@ -1,32 +1,35 @@
...
     <div class="pb-0.5">
-      <TextInput id="to" :model-value="formattedTo" :valid="toIsValid"
-      :placeholder="$t('InterfaceHomeSendModal.input1.placeholder')"
-      @input="onToInput" />
+      <TextInput
+        id="to"
+        :model-value="formattedTo"
+        :valid="toIsValid"
+        :placeholder="$t('InterfaceHomeSendModal.input1.placeholder')"
+        @input="onToInput"
```
Comment (cacampbell) [inline src/components/interface/TransferForm.vue:11]: its `@update:modelValue`

File: src/components/marketing/NavMenuButton.vue [hunk 11]
```diff
@@ -6,6 +6,10 @@
       rounded-md
       focus:outline-none
       focus:ring-2 focus:ring-inset focus:ring-white
+<<<<<<< HEAD
```
Comment (cacampbell) [inline src/components/marketing/NavMenuButton.vue:9]: Fix

File: src/domain/wallet/hardware-ledger.ts [hunk 12]
```diff
@@ -1,4 +1,8 @@
 import { Wallet } from "./abstract";
+<<<<<<< HEAD
```
Comment (cacampbell) [inline src/domain/wallet/hardware-ledger.ts:2]: Fix

File: src/domain/wallet/hardware-ledger.ts [hunk 13]
```diff
@@ -84,10 +88,21 @@ export class LedgerHardwareWallet extends Wallet {
     return response;
   }
 
+>>>>>>> 1071461646dafa61a6e27f9d6450406957b0577a
   hasPrivateKey(): boolean {
     return false;
   }
 
+<<<<<<< HEAD
```
Comment (cacampbell) [inline src/domain/wallet/hardware-ledger.ts:96]: Fix

File: src/pages/Marketing.vue [hunk 14]
```diff
@@ -8,6 +8,10 @@
       to-white
       dark:to-black
       overflow-x-hidden
+<<<<<<< HEAD
```
Comment (cacampbell) [inline src/pages/Marketing.vue:11]: Fix

File: src/pages/interface/ConvertUnits.vue [hunk 15]
```diff
@@ -45,12 +47,12 @@
         <TextInput
           class="m-1"
           v-model="state.valueRight"
-          :modelValue="'1'"
+          :modelValue="1"
```
Comment (cacampbell) [inline src/pages/interface/ConvertUnits.vue:50]: you cannot bind model value twice

File: src/pages/interface/DownloadFile.vue [hunk 16]
```diff
@@ -0,0 +1,67 @@
...
+      downloadId: null,
+      transactionType = "downloadFile"
+    });
+
+    function download(): void {
+      openModal();
+    }
+
+    function openModal(): void {
+      nextTick((state.showFeeModal = true));
```
Comment (cacampbell) [inline src/pages/interface/DownloadFile.vue:58]: callback, 

`nextTick(() => do stuff)`

File: src/pages/interface/DownloadFile.vue [hunk 17]
```diff
@@ -0,0 +1,67 @@
...
+    }
+
+    function closeModal(): void {
+      nextTick((state.showFeeModal = false));
```
Comment (cacampbell) [inline src/pages/interface/DownloadFile.vue:62]: callback

File: src/router.ts [hunk 18]
```diff
@@ -75,11 +74,8 @@ const routes = [
       },
       {
         path: "/access/hardware/ledger",
-        component: AccessLedger,
```
Comment (cacampbell) [inline src/router.ts:78]: ??

File: src/utils/test-bignumber.js [hunk 19]
```diff
@@ -0,0 +1,136 @@
+import BigNumber from "bignumber.js";
```
Comment (cacampbell) [inline src/utils/test-bignumber.js:1]: this can be removed

File: locales/en.json (hunk 2 above)
Comment (SMITH1103) [reply locales/en.json:53]: Not in use, will delete.

File: package.json (hunk 4 above)
Comment (SMITH1103) [reply package.json:43]: Fixed, will push changes.

File: src/components/base/DropdownSelector.vue (hunk 5 above)
Comment (SMITH1103) [reply src/components/base/DropdownSelector.vue:23]: Fixed, will push changes.

File: src/components/interface/Headline.vue (hunk 6 above)
Comment (SMITH1103) [reply src/components/interface/Headline.vue:10]: Fixed, will push changes.

File: src/pages/interface/DownloadFile.vue (hunk 16 above)
Comment (SMITH1103) [reply src/pages/interface/DownloadFile.vue:58]: This is fixed on a different branch, I started working on a separate branch to handle the Download tool.

File: src/pages/interface/DownloadFile.vue (hunk 17 above)
Comment (SMITH1103) [reply src/pages/interface/DownloadFile.vue:62]: This is fixed on a different branch, I started working on a separate branch to handle the Download tool.

File: src/router.ts (hunk 18 above)
Comment (SMITH1103) [reply src/router.ts:78]: I'm not sure where that came from, that's not mine.

File: index.html (hunk 1 above)
Comment (SMITH1103) [reply index.html]: Fixed, will push changes.

**Comment by SMITH1103** [comment]
Made fixes, closing this pull request.
